### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by jdcaballerov**
Added  " " to the form definition

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -672,7 +672,7 @@ a ```ngClass``` in the decorators scope. If not set att all the default value
 is ```{ 'glyphicon': true, 'glyphicon-ok': hasSuccess(), 'glyphicon-remove': hasError() }```
 
 ex. displaying an asterisk on required fields
-```javascript
+```javascriptele
```
Comment (davidlgj) [inline docs/index.md]: Whats `javascriptele`?

**Comment by jdcaballerov** [comment]
Sorry @davidlgj , that's my mistake.

**Comment by davidlgj** [comment]
:D I thought it was some cool version of syntax highlighting that made it look better

**Comment by stramel** [comment]
@jdcaballerov @davidlgj I thought the same thing :laughing:

**Comment by jdcaballerov** [comment]
:laughing:

**Pull request merged**